KP5-Topic2/KP4-Topic2/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files